components/dada-execute/src/step/apply_op.rs

```diff
@@ -43,7 +43,10 @@ impl Stepper<'_> {
                 Op::EqualEqual => Ok(self.machine.our_value(lhs == rhs)),
                 _ => op_error(),
             },
-            (&ObjectData::Uint(lhs), &ObjectData::Uint(rhs)) => match op {
+            (&ObjectData::Uint(lhs), &ObjectData::Uint(rhs))
+            | (&ObjectData::Uint(lhs), &ObjectData::UnsuffixedInt(rhs))
+            | (&ObjectData::UnsuffixedInt(lhs), &ObjectData::Uint(rhs))
+            | (&ObjectData::UnsuffixedInt(lhs), &ObjectData::UnsuffixedInt(rhs)) => match op {
```

Comment:
debatable how two unsuffixed integers should behave... I think I'd like `5 - 3` to result in an unsuffixed integer and `3 - 5` to fail at runtime (underflow). What do we do now?

Reply:
Now `3 - 5` fails at runtime (underflow), but `5 - 3` results in an unsigned integer. A `value` of type u64 is implicitly converted to ObjectData::Uint when we call `self.machine.our_value(value)` 
https://github.com/dada-lang/dada/blob/547f5166f525cfa4bb61e9fa0cde29dbc5a406b1/components/dada-execute/src/machine.rs#L209-L235